src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Amazon Lambda Telemetry integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's Amazon App Runner monitoring integration: what data it reports, and how to enable it."
+---
+
+ # New Relic AWS Lambda Telemetry extension
+
+The New Relic AWS Lambda Telemetry extension allows users to send telemetry data straight from their lambda functions into New Relic, with no need to set up triggers or connect Amazon CloudWatch.Send your logs, events, metrics and traces by simply adding the extension as the layer and inputting your New Relic license key.
+
+## Installation
+
+1. In the AWS Console go to **Lambda** > **Functions** then choose the function you'd like to ingest data from.
+2. Select **Layers** > **Add Layer**.
+3. In **Choose a layer** select the **Specify a layer by providing the ARN** option.
+4. Put `arn:aws:lambda:us-west-2:533243300146:layer:python-telemetry-api-extension:<version>` in the field, then click **Add**.
+5. Go to **configuration**, select **Environment variables** and add **Key: LICENSE_KEY  Value: <var>LICENSE_KEY</var>**. 
+
+To verify that set up has been done correctly, click the **Test** button in **code** section. 
+
+## Find your data
+
+After the test has completed, you can find your data on New Relic by navigating to **[https://one.newrelic.com/](https://one.newrelic.com/)** > **Browse data** > then selecting one of 4 categories: **Logs, Events, Metrics or Traces**.
+
+### Logs:
+
+Logs can be filtered by specific attributes set. Click **Attributes** in left menu and then choose the attributes of interest. For example:
+
+```
+SOURCE > api.logs
+PLUGIN > TYPE > lambda extension
+AWS > EXTENSION > python-telemetry-api-extension
+```
+
+### Metrics:
+
+Metrics sent by the extension are prefixed with `aws.telemetry.lambda_ext.`. Currently there are 6 AWS lambda metrics:
+
+```
+'durationMs'
+'billedDurationMs'
+'initDurationMs'
+'memorySizeMB'
+'maxMemoryUsedMB'
+'producedBytes'
+```
+
+### Events
+
+To see event sent by this extension, in **Custom events** section look for `Lambda_Ext_` prefix. Currently there are
+twelve event types:
+
+```
+platform_initStart
+platform_initRuntimeDone
+platform_initReport
+platform_start
+platform_runtimeDone
+platform_fault
+platform_report
+platform_extension
+platform_telemetrySubscription
+platform_logsDropped
+function
+extension
+```
+
+### Traces
+
+Traces data are displayed in **Traces** tab. Search for `python-telemetry-api-extension' to filter for these traces.
```

Comment:
AwsLambdaExtension should replace python-telemetry-api-extension
I's say "to filter for the relevant traces."